CIScripts/Test/PesterLogger/RemoteLogCollector.ps1

```diff
@@ -0,0 +1,90 @@
+. $PSScriptRoot/PesterLogger.ps1
+
+function New-LogSource {
+    Param([Parameter(Mandatory = $true)] [string] $Path,
+          [Parameter(Mandatory = $false)] [PSSessionT[]] $Sessions)
+
+    return $Sessions | ForEach-Object {
+        @{
+            Session = $_
+            Path = $Path
+        }
```

Comment:
Perhaps make it a (data-only) class?

Reply:
Classes introduce problems with importing stuff, as I learned. I try to avoid them now.